### PR TITLE
Create a frequently asked questions section in the docs

### PR DESCRIPTION
**Pull request by mdmurbach**
I've gotten a few questions recently that would probably make sense to add as a FAQ page in the docs.

Started with "What method does impedance.py use for fitting equivalent circuit models?", other one's that come to mind:
- "How do I cite impedance.py?" (This one becomes easier after the JOSS paper)
- "How do I get setup to contribute to the project?" (Fork, branch, PR, potential rebase, etc.)

I think we should plan on leaving this PR open for a bit, so please feel free to suggest other FAQs or push to this branch if you want to answer one of the above.

### CODE REVIEW

**Comment by mdmurbach** [comment]
You can see the compiled docs on read the docs here: https://impedancepy.readthedocs.io/en/docs-faq/faq.html

**Comment by BGerwe** [comment]
I'll work on 
> - "How do I get setup to contribute to the project?" (Fork, branch, PR, potential rebase, etc.)

Are there any issues with including screenshots where appropriate?

**Comment by mdmurbach** [comment]
Sounds great @BGerwe 😄 thanks!

You can also keep that FAQ answer a short summary and point folks to the [CONTRIBUTING.md](https://github.com/ECSHackWeek/impedance.py/blob/master/CONTRIBUTING.md) file (screenshots might be helpful to add there if anything seems missing).

**Comment by mdmurbach** [comment]
Went ahead and added a short section on contributing and I think we're pretty much ready to merge this in! Feel free to make any changes/additions as you see fit, @BGerwe.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,27 @@ Several example notebooks are provided in the examples/ directory. Opening these
...
 Several examples can be found in the `examples/` directory (the [Fitting impedance spectra notebook](https://impedancepy.readthedocs.io/en/latest/examples/fitting_example.html) is a great place to start) and the documentation can be found at [impedancepy.readthedocs.io](https://impedancepy.readthedocs.io/en/latest/).
 
+## Citing impedance.py
+
+[![DOI](https://joss.theoj.org/papers/10.21105/joss.02349/status.svg)](https://doi.org/10.21105/joss.02349)
+
+If you use impedance.py in published work, please consider citing https://joss.theoj.org/papers/10.21105/joss.02349 as
+
+```bash
+@article{Murbach2020,
```
Comment (BGerwe) [inline README.md:52]: Is this supposed to render as a journal citation (ex: Murbach, M.D., et al., "impedance.py: A Python package for electrochemical impedance analysis". J. Open Source Soft., ...)?

File: README.md [hunk 2]
```diff
@@ -42,6 +42,27 @@ Several example notebooks are provided in the examples/ directory. Opening these
...
+  doi = {10.21105/joss.02349},
+  url = {https://doi.org/10.21105/joss.02349},
+  year = {2020},
+  publisher = {The Open Journal},
+  volume = {5},
+  number = {52},
+  pages = {2349},
+  author = {Matthew D. Murbach and Brian Gerwe and Neal Dawson-Elli and Lok-kun Tsui},
+  title = {impedance.py: A Python package for electrochemical impedance analysis},
```
Comment (BGerwe) [inline README.md:61]: `for` is rendering in red as if it's a for loop

**Comment by BGerwe** [comment]
Sorry for taking so long on this... Pending clarification about the citation style your changes look good .

While looking through this stuff I noticed my LinKK docs were pretty confusing. So I rewrote some stuff to *hopefully* clarify what's happening under the hood.

File: README.md (hunk 1 above)
Comment (mdmurbach) [reply README.md:52]: It's actually just the raw .bibtex that you could use directly (took inspiration from this citing section from Altair [here](https://github.com/altair-viz/altair#citing-altair) 😛)

File: README.md (hunk 2 above)
Comment (mdmurbach) [reply README.md:61]: Good eye 🎯 fixed in: https://github.com/ECSHackWeek/impedance.py/commit/9d8e2fdc7c92fcc4cd0528808a449875ecf0a861

**Comment by mdmurbach** [comment]
Thanks for the review @BGerwe! I made a few edits/reworded some of the LinKK notes and I'd be curious to get your thoughts.

**Review by BGerwe — APPROVED** [review]

**Comment by BGerwe** [comment]
That looks great! Thanks for rewording and reorganizing.

**Pull request merged**